23_networks_vlan_nested_proxmox_cloud-init_27-oct-2022/README.md

```diff
@@ -337,15 +417,23 @@ netplan apply
   - 10.22.0.1
   - ip=10.22.0.16/24,gw=10.22.0.1
 * Start ubuntu6
-* Install qemu-guest-agent
+* Install _qemu-guest-agent_:
+    
+    ```
+    sudo apt update
+    sudo apt upgrade
+    sudo apt install -y qemu-guest-agent
+    systemctl enable --now qemu-guest-agent
```

Comment:
why the very last command doesn't use sudo (here and above in analogous places)?

Reply:
added sudo